BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -612,8 +612,19 @@ private void ConvertTargetSpeakerTrigger(Entity trigger, Entity targetSpeaker)
 
 		private void ConvertTargetSpeaker(Entity targetSpeaker)
 		{
+			var noise = targetSpeaker["noise"];
+			if (noise.StartsWith("sound/"))
+			{
+				var removeString = "sound/";
+				var index = noise.IndexOf(removeString);
+				var trimmedString = (index < 0)
+					? noise
+					: noise.Remove(index, removeString.Length);
+				noise = trimmedString;
```

Comment:
You can replace this with `noise.Replace("sound/", "");`

Reply:
I was thinking of a potential cases of a soundpath like sound/mysound/beep.wav, only wanting to change the first folder and not the rest